lib/rubocop/cop/layout/closing_heredoc_indentation.rb

```diff
@@ -79,10 +79,11 @@ def empty_heredoc?(node)
         end
 
         def argument_indentation_correct?(node)
-          node.argument? &&
```

Comment:
This cop was relying on the [bug outlined above](https://github.com/rubocop-hq/rubocop/pull/6204/files#r212209666).